barpyrus/lemonbar.py

```diff
@@ -28,9 +29,12 @@ def __init__(self, geometry = None,
         command += '-a 100 -d -u 2'.split(' ')
         command += [ '-B', background  ]
         command += [ '-F', foreground  ]
+        command += [ '-o 0' ]
         command += [ '-f', font  ]
+        command += [ '-o 0' ]
         command += '-f -*-*-*-*-*-*-2-*-*-*-*-*-*-*'.split(' ')
         if symbol_font != None:
+            command += [ '-o -%d' % symbol_vert_offset ]
```

Comment:
The lemonbar I'm using here does not know the `-o` flag you were passing to it. Looking at  lemonbar's source on https://github.com/LemonBoy/bar/blob/master/lemonbar.c#L1461 it seems that its about the randr output, so I think it does not belong to the general code. If you need it, we can add an `additional_args=[]` as a parameter to `__init__` such that you can pass it around from your config.py

Reply:
Hmm, I see, so I was using `lemonbar-xft` with this option: https://github.com/drscream/lemonbar-xft/blob/xft-port/lemonbar.c#L1494

Reply:
I see, thanks! Since adding this is part of #11 I'd remove it from the present PR.